CMakeLists.txt

```diff
@@ -8,11 +8,21 @@ set(VERSION "1.2.8")
 option(ASM686 "Enable building i686 assembly implementation")
 option(AMD64 "Enable building amd64 assembly implementation")
 
-set(INSTALL_BIN_DIR "${CMAKE_INSTALL_PREFIX}/bin" CACHE PATH "Installation directory for executables")
-set(INSTALL_LIB_DIR "${CMAKE_INSTALL_PREFIX}/lib" CACHE PATH "Installation directory for libraries")
-set(INSTALL_INC_DIR "${CMAKE_INSTALL_PREFIX}/include" CACHE PATH "Installation directory for headers")
-set(INSTALL_MAN_DIR "${CMAKE_INSTALL_PREFIX}/share/man" CACHE PATH "Installation directory for manual pages")
-set(INSTALL_PKGCONFIG_DIR "${CMAKE_INSTALL_PREFIX}/share/pkgconfig" CACHE PATH "Installation directory for pkgconfig (.pc) files")
+if (NOT INSTALL_BIN_DIR)
+    set(INSTALL_BIN_DIR "${CMAKE_INSTALL_PREFIX}/bin")
```

Comment:
Even better would be to remove `${CMAKE_INSTALL_PREFIX}/` here completely. CMake will add it at install time for any relative path.